### DISCUSSION TITLE
Support shift-clicking for quick disposing of items

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#7486] [Enhancement]
As per Discord user `@creamgod45#5506`;

Can a setting node be set? Don't check Dispose put items (ex: Dispose.nocheck ....), it's really annoying to backtrack items
I want to throw something fast!! The procedure for checking items has caused the original quick placement function to fail to work properly!!

CMI Dispose function.
You can add config.yml to the configuration file of the CMI plugin, and add a Dispose.nocheckitem configuration node. Or a better way to set it up. I want to trash quickly, but your Dispose interface keeps checking items and backtracking for items that were placed too quickly. I hope you can add a setting node that does not check items.

![image](https://user-images.githubusercontent.com/28841349/210396062-0506cf25-8ab4-4c14-b09a-10b0a3b2447d.png)

https://user-images.githubusercontent.com/28841349/210396114-d57db88c-1805-45bf-b9d0-f22c74d5024c.mp4

**Comment by creamgod45** [comment]
good 😊😊😊

**Comment by DeEthria** [comment]
So I'm for it too. Throwing back items is really annoying

**Comment by Zrips** [comment]
Gets a fix with next CMILib and CMI update
